helm-charts/falcon-sensor/README.md

```diff
@@ -136,6 +136,20 @@ For a complete listing of configurable parameters, run the following command:
 helm show values crowdstrike/falcon-sensor
 ```
 
+### GKE Autopilot Configuration
+Due to the Daemonset pods being run as a privileged, the following additional manifest is required to be applied to the cluster before deployment:
+```
+apiVersion: auto.gke.io/v1
+kind: AllowlistSynchronizer
+metadata:
+  name: crowdstrike-synchronizer
+spec:
+  allowlistPaths:
+  - CrowdStrike/falcon-sensor/*
```

Comment:
It looks like Alphonse provided more specific paths in the other PR https://github.com/CrowdStrike/falcon-helm/pull/393/files#r2061982966
```
- CrowdStrike/falcon-sensor/cleanup/crowdstrike-falconsensor-cleanup-allowlist-v1.0.1.yaml
- CrowdStrike/falcon-sensor/deploy/crowdstrike-falconsensor-deploy-allowlist-v1.0.0.yaml
- CrowdStrike/falcon-sensor/deploy/crowdstrike-falconsensor-deploy-allowlist-v1.0.2.yaml
- CrowdStrike/falcon-sensor/falconctl/crowdstrike-falconsensor-falconctl-allowlist-v1.0.1.yaml
```

Reply:
@mr-jungchoi Let's do the wildcard for now. Once we start getting more allowlists we can pin a version or wildcard for version.  It's not really an issue now since the webhook only needs to check matches for a few templates.